Makefile

```diff
@@ -109,9 +103,13 @@ fix-lint: ## fix lint issues using autoflake, autopep8, and isort
 
 # TEST TARGETS
 
-.PHONY: test-pytest
-test-pytest: ## run all the tests using pytest
-	invoke pytest
+.PHONY: test-integration
```

Comment:
This should be called `test-unit` right? Below it is also invoking `integration` where it should be `unit`